README.md

```diff
@@ -16,9 +16,9 @@ This repository ("`Code - OSS`") is where we (Microsoft) develop the [Visual Stu
 
 [Visual Studio Code](https://code.visualstudio.com) is a distribution of the `Code - OSS` repository with Microsoft-specific customizations released under a traditional [Microsoft product license](https://code.visualstudio.com/License/).
 
-[Visual Studio Code](https://code.visualstudio.com) combines the simplicity of a code editor with what developers need for their core edit-build-debug cycle. It provides comprehensive code editing, navigation, and understanding support along with lightweight debugging, a rich extensibility model, and lightweight integration with existing tools.
+[Visual Studio Code](https://code.visualstudio.com) combines the simplicity of a code editor with what developers need for their core edit-build-debug cycle. It provides comprehensive code editing, navigation, understanding support, lightweight debugging, a rich extensibility model, and lightweight integration with existing tools.
```

Comment:
This is an incorrect fix, much like the other PRs you're sending today. "editing, navigation, and understanding support" are modifiers on "code", thus the two lists are broken up.